miniapps/plasma/cold_plasma_dielectric_solver.cpp

```diff
@@ -359,7 +356,7 @@ CPDSolver::CPDSolver(ParMesh & pmesh, int order, double omega,
    // Define compatible parallel finite element spaces on the parallel
    // mesh. Here we use arbitrary order H1, Nedelec, and Raviart-Thomas finite
    // elements.
-   // H1FESpace_    = new H1_ParFESpace(pmesh_,order,pmesh_->Dimension());
+   H1FESpace_    = new H1_ParFESpace(pmesh_,order,pmesh_->Dimension());
```

Comment:
I was wondering about the order of approximation here. To be consistent with the first exact sequence spaces, should the RT space use `order-1` ?